src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/AddAppealResponseHandler.java

```diff
@@ -117,9 +118,13 @@ public boolean canHandle(
         asylumCase.write(RESPONDENT_DOCUMENTS, allRespondentDocuments);
 
         asylumCase.write(APPEAL_RESPONSE_AVAILABLE, YES);
-        asylumCase.write(REVIEW_RESPONSE_ACTION_AVAILABLE, YES);
+        asylumCase.write(REVIEW_RESPONSE_ACTION_AVAILABLE, NO);
         asylumCase.write(AMEND_RESPONSE_ACTION_AVAILABLE, YES);
 
+        asylumCase.write(REVIEW_HOME_OFFICE_RESPONSE_BY_LEGAL_REP, YES);
```

Comment:
Should we consider to schedule automatic direction for timed event service? It is a part of `requestResponseReview`
@Srihari-Nakkala @burale 
Automatic direction will not happen unless `requestResponseeReview` is triggered, which that event excludes from the flow. What do you think?

Reply:
@burale did you clarify with Pav and Srihari about sending automatic direction

Reply:
It is clarified with Pav and automatic direction for timed event added.